tools/ci_build/github/azure-pipelines/templates/react-native-ci.yml

```diff
@@ -251,8 +264,23 @@ stages:
       displayName: Pod install for onnxruntime react native ios e2e tests
 
     - script: |
-        keytool -genkey -v -keystore debug.keystore -alias androiddebugkey -storepass android \
-          -keypass android -keyalg RSA -keysize 2048 -validity 999999 -dname "CN=Android Debug,O=Android,C=US"
+        yarn add "jest@^29" --dev
+      workingDirectory: '$(Build.SourcesDirectory)/js/react_native/e2e'
+      displayName: yarn add jest for detox integration in js/react_native/e2e
+  
+    - script: |
+        yarn add detox --dev
+      workingDirectory: '$(Build.SourcesDirectory)/js/react_native/e2e'
+      displayName: install detox js/react_native/e2e
```

Comment:
do we still need these steps or would jest and detox be installed by the `yarn` call above?